2024-06-18 11:47:16,112 INFO    StreamThr :32936 [internal.py:wandb_internal():89] W&B internal server running at pid: 32936, started at: 2024-06-18 11:47:16.111851
2024-06-18 11:47:16,113 DEBUG   HandlerThread:32936 [handler.py:handle_request():144] handle_request: status
2024-06-18 11:47:16,119 INFO    WriterThread:32936 [datastore.py:open_for_write():85] open: D:\Research-Projects-jupyter\PFLlib\system\wandb\run-20240618_114716-txph0v8s\run-txph0v8s.wandb
2024-06-18 11:47:16,120 DEBUG   SenderThread:32936 [sender.py:send():379] send: header
2024-06-18 11:47:16,212 DEBUG   SenderThread:32936 [sender.py:send():379] send: run
2024-06-18 11:47:19,366 DEBUG   HandlerThread:32936 [handler.py:handle_request():144] handle_request: status_report
2024-06-18 11:47:21,260 DEBUG   HandlerThread:32936 [handler.py:handle_request():144] handle_request: keepalive
2024-06-18 11:47:24,420 DEBUG   HandlerThread:32936 [handler.py:handle_request():144] handle_request: status_report
2024-06-18 11:47:26,316 DEBUG   HandlerThread:32936 [handler.py:handle_request():144] handle_request: keepalive
2024-06-18 11:47:29,480 DEBUG   HandlerThread:32936 [handler.py:handle_request():144] handle_request: status_report
2024-06-18 11:47:31,358 DEBUG   HandlerThread:32936 [handler.py:handle_request():144] handle_request: keepalive
2024-06-18 11:47:34,522 DEBUG   HandlerThread:32936 [handler.py:handle_request():144] handle_request: status_report
2024-06-18 11:47:36,420 DEBUG   HandlerThread:32936 [handler.py:handle_request():144] handle_request: keepalive
2024-06-18 11:47:39,579 DEBUG   HandlerThread:32936 [handler.py:handle_request():144] handle_request: status_report
2024-06-18 11:47:40,478 INFO    SenderThread:32936 [dir_watcher.py:__init__():211] watching files in: D:\Research-Projects-jupyter\PFLlib\system\wandb\run-20240618_114716-txph0v8s\files
2024-06-18 11:47:40,478 INFO    SenderThread:32936 [sender.py:_start_run_threads():1124] run started: txph0v8s with start time 1718682436.111851
2024-06-18 11:47:40,478 DEBUG   HandlerThread:32936 [handler.py:handle_request():144] handle_request: status_report
2024-06-18 11:47:41,473 DEBUG   HandlerThread:32936 [handler.py:handle_request():144] handle_request: keepalive
2024-06-18 11:47:44,625 DEBUG   HandlerThread:32936 [handler.py:handle_request():144] handle_request: status_report
2024-06-18 11:47:45,525 DEBUG   HandlerThread:32936 [handler.py:handle_request():144] handle_request: status_report
2024-06-18 11:47:46,522 DEBUG   HandlerThread:32936 [handler.py:handle_request():144] handle_request: keepalive
2024-06-18 11:47:49,679 DEBUG   HandlerThread:32936 [handler.py:handle_request():144] handle_request: status_report
2024-06-18 11:47:50,584 DEBUG   HandlerThread:32936 [handler.py:handle_request():144] handle_request: status_report
2024-06-18 11:47:51,571 DEBUG   HandlerThread:32936 [handler.py:handle_request():144] handle_request: keepalive
2024-06-18 11:47:54,732 DEBUG   HandlerThread:32936 [handler.py:handle_request():144] handle_request: status_report
2024-06-18 11:47:55,627 DEBUG   HandlerThread:32936 [handler.py:handle_request():144] handle_request: status_report
2024-06-18 11:47:56,619 DEBUG   HandlerThread:32936 [handler.py:handle_request():144] handle_request: keepalive
2024-06-18 11:47:59,773 DEBUG   HandlerThread:32936 [handler.py:handle_request():144] handle_request: status_report
2024-06-18 11:48:00,664 DEBUG   HandlerThread:32936 [handler.py:handle_request():144] handle_request: status_report
2024-06-18 11:48:01,653 DEBUG   HandlerThread:32936 [handler.py:handle_request():144] handle_request: keepalive
2024-06-18 11:48:04,826 DEBUG   HandlerThread:32936 [handler.py:handle_request():144] handle_request: status_report
2024-06-18 11:48:05,714 DEBUG   HandlerThread:32936 [handler.py:handle_request():144] handle_request: status_report
2024-06-18 11:48:06,703 DEBUG   HandlerThread:32936 [handler.py:handle_request():144] handle_request: keepalive
2024-06-18 11:48:09,885 DEBUG   HandlerThread:32936 [handler.py:handle_request():144] handle_request: status_report
2024-06-18 11:48:10,723 WARNING FileStreamThread:32936 [file_stream.py:request_with_retry():671] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/paddeyzhang/SCL_test/txph0v8s/file_stream (Caused by ProxyError('Unable to connect to proxy', SSLError(SSLError(1, '[SSL: KRB5_S_TKT_NYV] unexpected eof while reading (_ssl.c:1091)')))). func: functools.partial(<bound method Session.post of <requests.sessions.Session object at 0x000002DE85794F08>>, timeout=60.0), args: ('https://api.wandb.ai/files/paddeyzhang/SCL_test/txph0v8s/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2024-06-18 11:48:10,751 DEBUG   HandlerThread:32936 [handler.py:handle_request():144] handle_request: status_report
2024-06-18 11:48:11,744 DEBUG   HandlerThread:32936 [handler.py:handle_request():144] handle_request: keepalive
